_data/projects/leaflet.yml

```diff
@@ -0,0 +1,10 @@
+name: Leaflet
+desc: JavaScript library for mobile-friendly interactive maps
+site: http://leafletjs.com
+tags:
+- Javascript
+- Maps
+- GIS
+upforgrabs:
+  name: easy-fix help-wanted
```

Comment:
When we're parsing this content to generate the site, we expect to have just one value here.

I'm not sure which one you're more fond of, but if you could cut it back to one here it'd be :heart: